cli/pkg/auth/utils.go

```diff
@@ -0,0 +1,14 @@
+package auth
+
+const (
+	FormMediaType = "application/x-www-form-urlencoded"
+	JSONMediaType = "application/json"
+)
+
+// OAuthTokenResponse contains the information returned after fetching an access token from the OAuth server.
+type OAuthTokenResponse struct {
```

Comment:
nit: Consider changing the package name to `oauth`. Then you could remove the `OAuth` prefix and the type name could just be `oauth.TokenResponse`.